templates/cc/msg.go

```diff
@@ -83,13 +83,4 @@ bool Validate(const {{ class . }}& m, pgv::ValidationMsg* err) {
 	return true;
 {{ end -}}
 }
-
```

Comment:
Why were these removed?

Reply:
I removed these because we already have hostname validation without including another template.